src/OpenTelemetry.Api/Tags/ITagContext.cs

```diff
@@ -20,7 +20,7 @@ namespace OpenTelemetry.Tags
     /// <summary>
     /// Collection of tags representing the tags context.
     /// </summary>
-    public interface ITagContext : IEnumerable<Tag>
+    public interface ITagContext : IEnumerable<DistributedContextEntry>
```

Comment:
Are we renaming this too in a follow up PR?

Reply:
Yes. I tried to rename everything at once - it is an unmanageable mess of updated files. That is impossible to review. Increments looks better. Issue #117 tracking it.

Reply:
Agreed. Better to break it up. Thanks